src/options.js

```diff
@@ -760,7 +760,11 @@ exports.val = {
    * Setting an entry to `true` enables reading and writing to that variable.
    * Setting it to `false` will trigger JSHint to consider that variable
    * read-only.
-   *
+   * 
+   * This option accepts an object, like so:
+   * 
+   *   // jshint globals:{"React": false, "_": false}
```

Comment:
this code doesn't work.. you need this to set the option
```
/* globals a:false */
```

In addition this is documentation inside jshint for the option, which can be set inline, or through config, so I am not sure this is the appropriate place to mention the format of the inline form